src/main/kotlin/sh/hsp/techtree/SimpleCommandLineParser.kt

```diff
@@ -0,0 +1,7 @@
+package sh.hsp.techtree
+
+class SimpleCommandLineParser : CommandLineParser {
+    override fun run(args: List<String>, func: (inputFile: String) -> Unit) {
+        func(args[0])
```

Comment:
proste i fajne.
Sprawdź tylko, czy args[0] to na pewno argument, a nie np interpretator czy coś.